src/Driver/driver_configs.jl

```diff
@@ -32,22 +32,35 @@ Parses custom command line option for tuples of two integers.
 function ArgParse.parse_item(::Type{NTuple{2, Int}}, s::AbstractString)
 
     str_array = split(s, ",")
-    horizontal = parse(Int, str_array[1])
-    vertical = parse(Int, str_array[2])
+    int_1 = length(str_array) > 0 ? parse(Int, str_array[1]) : 0
+    int_2 = length(str_array) > 1 ? parse(Int, str_array[2]) : 0
```

Comment:
@kpamnany here I took care of the case in which a single integer is passed instead of two. In that case, a 0 is placed as the second entry of the tuple. I tested this for single stack configs, e.g., after adding `ClimateMachine.init(parse_clargs = true)` in `test/Utilities/SingleStackUtils/ssu_tests.jl`, 
```
julia --project test/Utilities/SingleStackUtils/ssu_tests.jl --domain-height 2 --nelems 10
```

and the relevant info printed when the test is run is:

```
Info: Establishing single stack configuration for SingleStackUtilsTest
│     precision               = Float64
│     horiz polynomial order  = 5
│     vert polynomial order   = 5
│     domain_min              = 0.00 m, 0.00 m, 0.00 m
│     domain_max              = 1.00 m, 1.00 m, 2.00 m
│     # vert elems            = 10
│     MPI ranks               = 1
│     min(Δ_horz)             = 0.12 m
└     min(Δ_vert)             = 0.02 m
```